app/lib/user_settings_decorator.rb

```diff
@@ -171,8 +171,11 @@ def boolean_cast_setting(key)
     ActiveModel::Type::Boolean.new.cast(settings[key])
   end
 
-  def integer_cast_setting(key)
-    ActiveModel::Type::Integer.new.cast(settings[key])
+  def integer_cast_setting(key, min = nil, max = nil)
+    i = ActiveModel::Type::Integer.new.cast(settings[key])
+    return min if !min.nil? && i < min
```

Comment:
You can still input "e" (without any modifications) to the input field, which makes this throw a "undefined method '<' for nil:NilClass" error.

Reply:
I've been just waiting for something like this to happen lmao

Reply:
I love messing with forms :D
It's fixed :+1: